intellij-plugin-structure/structure-teamcity-actions/src/main/kotlin/com/jetbrains/plugin/structure/teamcity/action/Validator.kt

```diff
@@ -27,14 +27,27 @@ internal fun validateTeamCityAction(descriptor: TeamCityActionDescriptor) = sequ
   validateSpecVersion(descriptor.specVersion, ActionSpecVersion.NAME, ActionSpecVersion.DESCRIPTION)
 
   validateExists(descriptor.name, ActionName.NAME, ActionName.DESCRIPTION)
-  validateNotEmptyIfExists(descriptor.name, ActionName.NAME, ActionName.DESCRIPTION)
-  validateMaxLength(descriptor.name, ActionName.NAME, ActionName.DESCRIPTION, ActionName.MAX_LENGTH)
+
+  validateExists(descriptor.getNamespace(), ActionName.Namespace.NAME, ActionName.Namespace.DESCRIPTION)
+  validateNotEmptyIfExists(descriptor.getNamespace(), ActionName.Namespace.NAME, ActionName.Namespace.DESCRIPTION)
```

Comment:
It's not great that the validation error message is phrased like `The property namespace...`, because the namespace is not an actual property on the YAML. However, this is not a major issue, and I think it can be fixed later in a different PR.